lib/webpacker/dev_server.rb

```diff
@@ -0,0 +1,73 @@
+# Same convention as manifest/configuration.rb
+
+# Loads webpacker configuration from config/webpacker.yml
+
+require "webpacker/configuration"
+
+class Webpacker::DevServer < Webpacker::FileLoader
+  class << self
+    def enabled?
+      case ENV["WEBPACKER_DEV_SERVER"]
```

Comment:
See my comments below on how different team members want "live-reloading" or just the statically built files.